src/ape/api/providers.py

```diff
@@ -722,6 +722,13 @@ def get_contract_logs(
         abis = abi if isinstance(abi, (list, tuple)) else [abi]
         for abi in abis:
             filter_data = {"address": address, **filter_args}
+
+            if "fromBlock" not in filter_args:
```

Comment:
Perhaps `filter_args` should be spelt-out kwargs here because otherwise we'd be mixing casings in the codebase

Reply:
What do you mean by `spelt-out kwargs`?

Like change the name of `filter_args` to `filter_kwargs` ?

Reply:
Ohhh I see what you mean now.
Yes, I agree!